src/components/Router/StepComponentMap.ts

```diff
@@ -130,7 +130,7 @@ const buildCaptureStepComponents = (
 
   return {
     welcome: [Welcome],
-    ...(SDK_ENV !== 'Auth' && { userConsent: [UserConsent] }),
+    userConsent: [UserConsent],
```

Comment:
I believe we wanted this to be optional since the legal text on this screen is US-focused?